### ISSUE TITLE
Request : Use the songs used on youtube music not on youtube

### ISSUE THREAD

**Issue by bronze7** [KRTirtho/spotube#1012]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

The songs in youtube have some speech in between songs, some of them seem rushed and omits some verses of the original song. SponsorBlock often skips the song part.

### Suggested solution

Youtube often hosts the original song for youtube music. If possible stream that part. I can use JioSaavn but it doesn't have big collection youtube has.

**Comment by 1n9i9c7om** [comment]
+1, especially if this allowed us to log in with a YouTube Music premium account to unlock the higher quality (should be 256 kbits AAC compared to 128 kbits AAC from what I've read?)

**Comment by bronze7** [comment]
> +1, especially if this allowed us to log in with a YouTube Music premium account to unlock the higher quality (should be 256 kbits AAC compared to 128 kbits AAC from what I've read?)

Yeah but the major reason people use this app is to avoid paywalls. I pay for spotify but I don't like the way it pushes free users by restricting basic features like shuffling songs in playlists and move the seek bar.
There is an app called BlackHole (https://github.com/Sangwan5688/BlackHole) which solves the issue I am refering to. but I found this app to seamlessly integrate with my spotify account so I can use the spotify algorithm and playlists.
